packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -65,11 +65,13 @@ const getColumnLookup = <T extends DomainObject>(): Record<
     key: 'name',
     label: 'label.name',
     width: 75,
+    sortable: false,
```

Comment:
The item list would be better with the name sortable as well as the code, I think

Reply:
Ah of course! I removed it thinking it was for the invoices list. I've since added the columns as sortable to the api in  #374